src/D2L.Security.OAuth2.TestFramework/D2L.Security.OAuth2.TestFramework.csproj

```diff
@@ -2,7 +2,7 @@
 	<Sdk Name="Microsoft.Build.CentralPackageVersions" />
 
 	<PropertyGroup>
-		<TargetFrameworks>net472;net48</TargetFrameworks>
+		<TargetFrameworks>net472</TargetFrameworks>

```

Comment:
Removed the extra net48 targets because they weren't really doing anything as far as I know (other than making the build slower)